### ISSUE TITLE
Improve error message for too large numbers

### ISSUE THREAD

**Issue by lasinicl** [ballerina-platform/ballerina-lang#32741]
**Description:**
$title

**Describe your problem(s)**
```ballerina
public function main() {
    int a = 9223372036854775808; 
}
```
The above program results in `Integer '9223372036854775808' too large` compilation error. Better if we can provide more clear, straight forward error message.

**Comment by MaryamZi** [comment]
@suleka96, this will be fixed in https://github.com/ballerina-platform/ballerina-lang/pull/34622, right?

**Comment by KavinduZoysa** [comment]
This is fixed in `Ballerina 2201.4.0`.
